charts/datadog/templates/install_info-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "datadog.fullname" . }}-installinfo
+  labels:
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/name: "{{ template "datadog.fullname" . }}"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+  annotations:
+    checksum/install_info: {{ printf "%s-%s" .Chart.Name .Chart.Version | sha256sum }}
+data:
+  install_info: |
+    ---
+    install_method:
+      tool: helm
+      tool_version: {{ .Release.Service }}
```

Comment:
From doc `.Release.Service` is always `Helm`. Not sure if we can get Helm version, otherwise maybe we should put `Capabilities.KubeVersion`

Reply:
I saw that in the doc. So, I wondered what the value would be with Spinnaker. (Given the fact that Spinnaker re-implements the Helm features without relying on it.)

`KubeVersion` can be interesting, but it’s not really linked to the installation method.